python-checks/src/test/resources/checks/unusedFunctionParameter/unusedFunctionParameter.py

```diff
@@ -161,3 +161,35 @@ def param_referenced_in_comment_no_issue_3(unused_param_2):
     def param_accessed_through_pandas_no_issue(sample_df, area_of_interest): # OK
         sample_df.query('area == @area_of_interest').population
         print("hello")
+
+import abc
+class Test(abc.ABC):
+    def execute_test(self, name):
+        print("Test")
+
```

Comment:
As we discussed, I'd like to see an additional test where the inheritance from `abc.ABC` is not direct (having another class that inherits from the current `Test` for example).
I expect that an issue will be raised in that case and I think it's important to have that behavior documented.